Makefile

```diff
@@ -60,16 +58,17 @@ help:
 # Build targets. Calling setup is all that is needed for the local files to be installed as needed. Bundesnetz may cause problem.
 
 python: build/python
-	@echo "Python installed"
+	@echo $(shell $(SYSTEM_PYTHON_CMD) --version 2>&1) "installed"
 
 .PHONY: setup
 setup: python .venv/build.timestamp
 
 
-local/bin/python3.7:
-	mkdir -p $(CURRENT_DIR)/local;
-	curl -z $(CURRENT_DIR)/local/Python-$(PYTHON_VERSION).tar.xz https://www.python.org/ftp/python/$(PYTHON_VERSION)/Python-$(PYTHON_VERSION).tar.xz -o $(CURRENT_DIR)/local/Python-$(PYTHON_VERSION).tar.xz;
-	cd $(CURRENT_DIR)/local && tar -xf Python-$(PYTHON_VERSION).tar.xz && Python-$(PYTHON_VERSION)/configure --prefix=$(CURRENT_DIR)/local/ && make altinstall
+$(PYTHON_LOCAL_DIR)/bin/python3.7:
+	@echo "Building a local python..."
+	mkdir -p $(PYTHON_LOCAL_DIR);
+	curl -z $(PYTHON_LOCAL_DIR)/Python-$(PYTHON_VERSION).tar.xz https://www.python.org/ftp/python/$(PYTHON_VERSION)/Python-$(PYTHON_VERSION).tar.xz -o $(PYTHON_LOCAL_DIR)/Python-$(PYTHON_VERSION).tar.xz;
```

Comment:
This won't compile on your old  Debian `Jessie` instances. Do you all have a docker instance or do you develop locally?

Reply:
I don't know what are the `Jessie` instances and when/where they are used ? but I'm also not very happy for the local python build as it also don't compile on a fresh ubuntu 18.04 installation (missing a lot of dependencies).